test/FunctionalTests/Microsoft.OData.Edm.Tests/Vocabularies/CoreVocabularyTests.cs

```diff
@@ -378,6 +384,19 @@ public void TestBaseCoreVocabularyModel()
             Assert.NotNull(isLanguageDependentType);
             Assert.Equal(EdmPrimitiveTypeKind.Boolean, isLanguageDependentType.UnderlyingType.PrimitiveKind);
 
+            var requiresExplicitBindingTerm = coreVocModel.FindTerm("Org.OData.Core.V1.RequiresExplicitBinding");
+            Assert.NotNull(requiresExplicitBindingTerm);
+            var requiresExplicitBindingType = requiresExplicitBindingTerm.Type.Definition as IEdmTypeDefinition;
+            Assert.NotNull(requiresExplicitBindingType);
+            Assert.Equal(EdmPrimitiveTypeKind.Boolean, requiresExplicitBindingType.UnderlyingType.PrimitiveKind);
+
+            var explicitOperationBindingsTerm = coreVocModel.FindTerm("Org.OData.Core.V1.ExplicitOperationBindings");
+            Assert.NotNull(explicitOperationBindingsTerm);
+            var explicitOperationBindingsType = explicitOperationBindingsTerm.Type.Definition;
+            Assert.NotNull(explicitOperationBindingsType);
+            Assert.Equal("Collection(Org.OData.Core.V1.QualifiedBoundOperationName)", explicitOperationBindingsType.FullTypeName());

```

Comment:
>QualifiedBoundOperationName [](http://example.com/codeflow?start=55&length=27)

Once you add the QualifiedBoundOperationName, you'll want to add a test to retrieve and validate that term and then verify that explicitOperationBindingsType is of that type.